src/btree/bt_debug.c

```diff
@@ -1267,8 +1268,7 @@ __debug_cell(WT_DBG *ds, const WT_PAGE_HEADER *dsk, WT_CELL_UNPACK *unpack)
 	WT_DECL_ITEM(buf);
 	WT_DECL_RET;
 	WT_SESSION_IMPL *session;
-	char hex_ts[3][WT_TS_HEX_SIZE];
-	const char *type;
```

Comment:
My only comment on this is if the `type` is really not useful in debugging. It seems like removing the type is both a gratuitous change and probably should not be folded into the timestamp work.

Reply:
I believe the `type` is duplicated information, compare the output of the initial output lines in `__debug_cell`:
```
   1275         WT_RET(ds->f(ds, "\t%s: len %" PRIu32,
   1276             __wt_cell_type_string(unpack->raw), unpack->size));
```
and the `type` assignments that were removed.

Let me know if you want this change pulled out into a separate ticket.

Reply:
Thanks for the explanation. I only viewed the diff. I agree it is duplicated.